Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.11-alpine
+
+RUN pip install requests
+
+ADD . /src
+
+# UDP
+EXPOSE 1812
```

Comment:
Using "EXPOSE 1812/UDP" might eliminate the need for "-net" networking. Docker docs are a little slim.